collect_app/src/main/java/org/odk/collect/android/utilities/DateTimeUtils.java

```diff
@@ -125,4 +125,29 @@ public static DatePickerDetails getDatePickerDetails(String appearance) {
 
         return new DatePickerDetails(datePickerType, datePickerMode);
     }
+
+    /**
+     * Converts milliseconds time to Hours:Minutes:Seconds format
+     */
+    public static String millisToTimer(long milliseconds) {
+        StringBuilder sb = new StringBuilder();
+
```

Comment:
I suggest you remove this method and have the caller use DateTimeFormatter instead.

Reply:
@dcbriccetti  Done. 
I used `new DateTime(millis, DateTimeZone.UTC).toString("mm:ss");` this to convert `millis` to `mm:ss`

Reply:
Oh, even better than what I was thinking!